src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -396,6 +400,15 @@ public sealed class AdoConnector {
 
                 if (string.Equals(field, "System.Tags")) {
                     value += ";Onefuzz";
+                } else if (string.Equals(field, "System.Title") && value.Length > MAX_SYSTEM_TITLE_LENGTH) {
+                    var titleHashString = System.Security.Cryptography.SHA256.HashData(Encoding.UTF8.GetBytes(value))
+                        .Aggregate(
```

Comment:
You can use `Convert.ToHexString` here

Reply:
To replace the hash? Wouldn't that risk collision for titles starting with the same first 128 chars?

Reply:
To replace the conversion to hex:

```cs
hash.Aggregate(
  new StringBuilder(),
  (builder, next) => builder.Append(next.ToString("x2")),
  builder => builder.ToString())
```
  
Is equivalent to (aside from being in a different case):
```cs
Convert.ToHexString(hash)
```

Reply:
Thanks for the clarification! I probably should have been able to figure that `ToHexString` would be for the hex string conversion part.